build.gradle

```diff
@@ -82,9 +82,8 @@ dependencies {
     }
     embed "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
     embed "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
-    
-    embed 'com.github.jagrosh:DiscordIPC:master-SNAPSHOT' // old discord library, but it has Json classes that we use in StaffUtils, so don't remove it unless you replace it
```

Comment:
guessing you did something with the json
as long as it works im not complaining

Reply:
check staffutils